hedera-node/hedera-app/src/test/java/com/hedera/node/app/fees/MonoFeeAccumulatorTest.java

```diff
@@ -47,32 +52,51 @@ class MonoFeeAccumulatorTest {
     @Mock
     private StateView stateView;
 
+    @Mock
+    private MonoGetTopicInfoUsage getTopicInfoUsage;
+
+    @Mock
+    private ReadableStoreFactory readableStoreFactory;
+
+    @Mock
+    private ReadableTopicStore readableTopicStore;
+
     private MonoFeeAccumulator subject;
 
     @BeforeEach
     void setUp() {
-        subject = new MonoFeeAccumulator(usageBasedFeeCalculator, usagePricesProvider, () -> stateView);
+        subject = new MonoFeeAccumulator(
+                usageBasedFeeCalculator, getTopicInfoUsage, usagePricesProvider, () -> stateView);
+    }
+
+    @Test
+    void usesMonoAdapterDirectlyForGetTopicInfo() {
```

Comment:
Should this be kept commented?

Reply:
I think relative to the overall "compilability" of the branch, this test crosses the un-commented bar. 😁